projects/canopy/src/lib/pagination/pagination.component.scss

```diff
@@ -0,0 +1,42 @@
+@import '../../styles/mixins';
+
+.lg-pagination {
+  ul {
+    list-style-type: none;
+    display: flex;
+    flex-direction: row;
+    margin: 0;
+  }
+
+  /* stylelint-disable */
+  --button-size: 44px;
+  /* stylelint-enable */
+
+  .lg-pagination__button {
+    width: var(--button-size);
+    height: var(--button-size);
+    border-width: 0;
+    border-radius: 50%;
+    font-size: var(--text-fs--8-size);
+    background-color: transparent;
+
+    &:hover:not(:disabled) {
+      background-color: var(--color-charcoal);
+      color: var(--color-white);
+    }
+
+    &:focus,
+    &:hover:focus,
+    &:hover:focus:hover {
+      @include lg-focus-outline(
+        var(--default-focus-color),
```

Comment:
There shouldn't be any need for passing the two values to the `lg-focus-outline` as they are the default ones 🙂

Reply:
Good point! Thanks for spotting